src/test/controller/UserController.test.ts

```diff
@@ -222,9 +222,57 @@ describe('Unit tests for UserController', function () {
       });
   });
 
-  test('DELETE /users/:id', async (done) => {
+  test('DELETE /users/:id - ForbiddenError should be caught', async (done) => {
```

Comment:
Maybe we can make the title less technical like:
```suggestion
  test('DELETE /users/:id - Should not be able to delete another user', async (done) => {
```